pkg/controller/secretannotator/constants/const.go

```diff
@@ -0,0 +1,25 @@
+package constants
+
+const (
+	ControllerName = "secretannotator"
+
+	// TODO: dynamically detect which environment we're running on
+	CloudCredSecretName      = "aws-creds"
```

Comment:
this const can likely go, as it should only be down in the new aws secret annotator controller and actuator